buildenv/jenkins/JenkinsfileBase

```diff
@@ -373,7 +373,8 @@ def setup() {
 			setupEnv()
 
 			if (params.SDK_RESOURCE == 'nightly' && params.CUSTOMIZED_SDK_URL) {
-				CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"
+				// remove single quote to allow variables to be set in CUSTOMIZED_SDK_URL
+				CUSTOMIZED_SDK_URL_OPTION = "-c ${params.CUSTOMIZED_SDK_URL}"
 			} else if (params.CUSTOMIZED_SDK_URL) {
 				SDK_RESOURCE = "customized"
 				CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"
```

Comment:
Should this line also updated with single quote removed?

Reply:
No, we cannot. The purpose of having a single quote is for multiple URLs (JDK, JRE, test image, etc) in CUSTOMIZED_SDK_URL value. 
This is PR is for the special case - nightly with artifactory link. In this case, only one artifactory URL is expected, so it is okay to remove the single quote.